Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -679,6 +679,10 @@ def generate_header(serialized_types, serialized_enums, additional_forward_decla
     for header in ['<wtf/ArgumentCoder.h>', '<wtf/OptionSet.h>', '<wtf/Ref.h>', '<wtf/RetainPtr.h>']:
         result.append(f'#include {header}')
 
+    result.append(f'#if USE(CF)')
+    result.append(f'#include <Security/SecTrust.h>')
```

Comment:
Please run make -C Source/WebKit/Scripts/webkit/tests when changing this.
Also, this is suspicious.  It looks like it'll include a header from a widely-included header.  Can this be avoided?